src/util/index.js

```diff
@@ -33,6 +36,47 @@ export function buildContent(options : ContentConfig) : string {
     return content || '';
 }
 
+export function getBuyerCountry(fundingOption : string) : string {
+    switch (fundingOption) {
+    case 'PayUponInvoice': {
+        return 'DE';
+    }
+    case 'boletobancario': {
+        return 'BR';
+    }
+    default: {
+        return 'US';
```

Comment:
better return null as default. We can set the default country by use case when we call this function. But the function should be generic.

Reply:
Sure can fix this